src/Public/js/nipsweb.managelibrary.js

```diff
@@ -67,52 +72,63 @@ var Library = function () {
                 paramname: 'audio',
                 error: filedrop_error_handler,
                 allowedfiletypes: allowed_mp3,
-                maxfiles: 20,
+                maxfiles: 1,
                 maxfilesize: mConfig.audio_upload_max_size,
                 queuefiles: 1,
                 drop: function () {
-                    $('#central-status').html('Reading file (0%)...');
+                    $('#central-status').html(icon_loading + 'Reading file (0%)...');
                 },
                 uploadStarted: function (i, file, total) {
-                    $('#central-status').html('Uploading ' + file.name + '... (' + byteSize(file.size) + ')');
+                    $('#central-status').html(icon_loading + 'Uploading ' + file.name + '... (' + byteSize(file.size) + ')');
                 },
                 progressUpdated: function (i, file, progress) {
-                    $('#central-status').html('Reading ' + file.name + ' (' + progress + '%)...');
+                    $('#central-status').html(icon_loading + 'Reading ' + file.name + ' (' + progress + '%)...');
                 },
                 uploadFinished: function (i, file, response, time) {
-                    var status = 'Uploaded ' + file.name;
+                    var status = icon_ok + 'Uploaded ' + file.name;
                     $('#central-status').html(status);
 
                     setTimeout(
                         function () {
                             if ($('#central-status').html() == status) {
-                                $('#central-status').html('Ready');
+                                $('#central-status').html(icon_ok + 'Ready');
                             }
                         },
                         5000
                     )
 
                     var manual_track = false;
-                    if (response['status'] !== 'OK' || response.analysis.length === 0) {
-                        var manual_div = document.getElementById('track-manual-entry');
+                    var manual_div = document.getElementById('track-manual-entry');
                         if (manual_div !== null) {
-                            // If the div exists, then the user has permission to upload a track
-                            // manually, so display the div and set manual_track to true.
-                            $(manual_div).show();
-                            manual_track = true;
-                        }
-                    }
-
+                            if (response['status'] !== 'OK' && response['submittable'] == true ) {
+                                
+                                    // If the div exists, then the user has permission to upload a track
```

Comment:
formatting (among others) JS is also 2-spaces

I know several existing JS files don't follow that, but that's for another time..